### PR TITLE
💄 Add featured NFT stories

### PR DESCRIPTION
**Pull request by nwingt**
image

### CODE REVIEW

**Review by williamchong — CHANGES_REQUESTED** [review]
Why huge diff in locale jsons?

**Comment by nwingt** [comment]
> Why huge diff in locale jsons?

Resolved

File: src/locales/en.json [hunk 1]
```diff
@@ -284,6 +284,12 @@
   "nft_details_page_title": "Writing NFT",
   "nft_details_page_title_collector": "Collectors",
   "nft_details_page_title_transfer": "Transfer Your NFT",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_description": "在這人人都可以成為創作者的年代，內容轉向數位，卻仍然沒有一個可以跨越平台限制，自由買賣字的地方。區塊勢以最多人訂閱的中文區塊鏈媒體的角度，紀錄了全球第一家可以閱讀自由，創作自主之地，請進入「無人書店」。",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_title": "歡近來到全球第一家「無人書店」🔮",
+  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_description": "「這週大部分時間身處加州海岸的紅樹林，容許我偷懶一下，長信欠奉，附上明信片一張，閒話日常⋯⋯」2022 年初夏，k 先生凌晨 5 點多坐在舊金山的偌大教堂的長椅，兩層樓高的圓拱大窗，逐漸透出晨光第一線，給遠方友人寫下了片言隻語。這帶有獨一無二郵戳的明信片，等著你去領取。",
+  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_title": "你有一封未讀的新郵件 📩",
+  "nft_featured_likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc_description": "只需要帶著一部電腦到有網路的地方，便可以在世界各地任何一個地方持續創作。數位移民來得簡單實在，來一場説走就走的旅程吧。",
+  "nft_featured_likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc_title": "你的數位遊牧嚮導 👾",
```
Comment (williamchong) [inline src/locales/en.json:292]:
```suggestion
  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_description": "Everyone is a content creator in this modern digital era. However different limitations still exist on internet platforms, there is no one place in which words and writings can be flown freely. Blocktrend, the most subscribed chinese blockchain media, recorded the story of the first bookstore that restore freedom and ownership to readers and authors. Welcome to the "Web3 Bookstore"",
  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_title": "Welcome to the first "Web3 Bookstore" in the world🔮",
  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_description": "「這週大部分時間身處加州海岸的紅樹林，容許我偷懶一下，長信欠奉，附上明信片一張，閒話日常⋯⋯」It's 5 am in an early Summer day in 2022, on the long bench under the church in SF sits Mr K. When the first glimpse of sunlight shines through the arcaded windows, Mr K jotted down the words to his distant friends. This unique postcard is now awaiting for your collection.",
  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_title": "You have a new mail 📩",
  "nft_featured_likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc_description": "A laptop and wifi is all it takes, journey across the globe while creating content. Explore the simple yet solid life of digital nomads.",
  "nft_featured_likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc_title": "Your digital nomad guide👾",

```
simple translation just to make ppl complain and improve

**Review by williamchong — CHANGES_REQUESTED** [review]

**Review by AuroraHuang22 — APPROVED** [review]

File: src/locales/en.json [hunk 2]
```diff
@@ -284,6 +284,13 @@
...
   "nft_details_page_title_collector": "Collectors",
   "nft_details_page_title_transfer": "Transfer Your NFT",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_description": "Everyone is a content creator in this modern digital era. However different limitations still exist on internet platforms, there is no one place in which words and writings can be flown freely. Blocktrend, the most subscribed chinese blockchain media, recorded the story of the first bookstore that restore freedom and ownership to readers and authors. Welcome to the \"Web3 Bookstore\"",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_title": "Welcome to the first \"Web3 Bookstore\" in the world🔮",
+  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_description": "「這週大部分時間身處加州海岸的紅樹林，容許我偷懶一下，長信欠奉，附上明信片一張，閒話日常⋯⋯」It's 5 am in an early Summer day in 2022, on the long bench under the church in SF sits Mr K. When the first glimpse of sunlight shines through the arcaded windows, Mr K jotted down a few words to his distant friends. This unique postcard is now awaiting for you.",
```
Comment (phoebepoon) [inline src/locales/en.json:289]: It's 5 am in an early summer day in 2022, at the long bench under the church in SF sits Mr K. When the first glimpse of sunlight shines through the arcaded windows, Mr.K jotted down a few words to his distant friends. It is a unique postcard now waiting for you to collect.

File: src/locales/en.json [hunk 3]
```diff
@@ -284,6 +284,13 @@
...
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_title": "Welcome to the first \"Web3 Bookstore\" in the world🔮",
+  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_description": "「這週大部分時間身處加州海岸的紅樹林，容許我偷懶一下，長信欠奉，附上明信片一張，閒話日常⋯⋯」It's 5 am in an early Summer day in 2022, on the long bench under the church in SF sits Mr K. When the first glimpse of sunlight shines through the arcaded windows, Mr K jotted down a few words to his distant friends. This unique postcard is now awaiting for you.",
+  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_title": "You have a new mail 📩",
+  "nft_featured_likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc_description": "A laptop and wifi is all it takes, journey across the globe while creating content. Explore the simple yet solid life of digital nomads.",
```
Comment (phoebepoon) [inline src/locales/en.json:291]: All you need is a laptop and an internet connection to become a digital nomads. Explore the world while creating content for a living. Let's get started here.

Comment (williamchong) [inline src/locales/en.json:291]:
```suggestion
  "nft_featured_likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc_description": "All you need is a laptop and an internet connection to become a digital nomads. Explore the world while creating content for a living. Let's get started here.",

```

File: src/locales/en.json (hunk 2 above)
Comment (williamchong) [inline src/locales/en.json:289]:
```suggestion
  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_description": "It's 5 am in an early summer day in 2022, at the long bench under the church in SF sits Mr K. When the first glimpse of sunlight shines through the arcaded windows, Mr.K jotted down a few words to his distant friends. It is a unique postcard now waiting for you to collect.",

```

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/locales/en.json (hunk 2 above)
Comment (williamchong) [reply src/locales/en.json:289]: Remove the chinese part?

File: src/locales/en.json [hunk 4]
```diff
@@ -284,6 +284,13 @@
   "nft_details_page_title": "Writing NFT",
   "nft_details_page_title_collector": "Collectors",
   "nft_details_page_title_transfer": "Transfer Your NFT",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_description": "Everyone is a content creator in this modern digital era. However different limitations still exist on internet platforms, there is no one place in which words and writings can be flown freely. Blocktrend, the most subscribed chinese blockchain media, recorded the story of the first bookstore that restore freedom and ownership to readers and authors. Welcome to the \"Web3 Bookstore\"",
```
Comment (phoebepoon) [inline src/locales/en.json:287]: We are all content creators in this modern digital era. However, there isn't a place where words can be bought and sold freely beyond platform restrictions. Blocktrend, the most subscribed Chinese Blockchain media recorded the story of the first Web3 bookstore that restored freedom and ownership to readers and writers. The world's first bookstore where you can read freely and create independently. Please enter the "Unmanned Bookstore".

**Review by williamchong — APPROVED** [review]

File: src/locales/en.json [hunk 5]
```diff
@@ -284,6 +284,15 @@
...
```
Comment (williamchong) [inline src/locales/en.json:287]:
```suggestion
  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_description": 
We are all content creators in this modern digital era. However, there isn't a place where words can be bought and sold freely beyond platform restrictions. Blocktrend, the most subscribed Chinese Blockchain media recorded the story of the first Web3 bookstore that restored freedom and ownership to readers and writers. The world's first bookstore where you can read freely and create independently. Please enter the \"Unmanned Bookstore\".",
```

**Review by williamchong — CHANGES_REQUESTED** [review]

**Review by williamchong — APPROVED** [review]

File: src/locales/en.json [hunk 6]
```diff
@@ -284,6 +284,15 @@
   "nft_details_page_title": "Writing NFT",
   "nft_details_page_title_collector": "Collectors",
   "nft_details_page_title_transfer": "Transfer Your NFT",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_description": "We are all content creators in this modern digital era. However, there isn't a place where words can be bought and sold freely beyond platform restrictions. Blocktrend, the most subscribed Chinese Blockchain media recorded the story of the first Web3 bookstore that restored freedom and ownership to readers and writers. The world's first bookstore where you can read freely and create independently. Please enter the \"Unmanned Bookstore\".",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_title": "Welcome to the first \"Web3 Bookstore\" in the world🔮",
```
Comment (nwingt) [inline src/locales/en.json:288]:
```suggestion
  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_title": "Welcome to the first \"Web3 Bookstore\" in the world 🔮",

```

File: src/locales/en.json [hunk 7]
```diff
@@ -284,6 +284,15 @@
...
+  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_description": "It's 5 am in an early summer day in 2022, at the long bench under the church in SF sits Mr K. When the first glimpse of sunlight shines through the arcaded windows, Mr.K jotted down a few words to his distant friends. It is a unique postcard now waiting for you to collect.",
+
+  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_title": "You have a new mail 📩",
+  "nft_featured_likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc_description": "All you need is a laptop and an internet connection to become a digital nomads. Explore the world while creating content for a living. Let's get started here.",
+
+  "nft_featured_likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc_title": "Your digital nomad guide👾",
```
Comment (nwingt) [inline src/locales/en.json:294]:
```suggestion
  "nft_featured_likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc_title": "Your digital nomad guide 👾",

```

**Pull request merged**